### PR TITLE
feat(theme): add currentColor, inherit, and transparent props to colors

### PR DESCRIPTION
**Pull request by jorgemoya**
Add inherit and transparent props to colors object for use in BD components.

### CODE REVIEW

**Comment by chanceaclark** [comment]
Can we add a description for this PR?

**Comment by jorgemoya** [comment]
> Can we add a description for this PR?

Done!

File: packages/big-design-theme/src/system/colors.ts [hunk 1]
```diff
@@ -49,6 +52,9 @@ export interface Colors {
 }
 
 export const colors: Colors = {
+  inherit: 'inherit',
+  transparent: 'transparent',
```
Comment (chanceaclark) [inline packages/big-design-theme/src/system/colors.ts:56]: 🍹 What do you think about adding `currentColor` as well?

**Review by chanceaclark — APPROVED** [review]

File: packages/big-design-theme/src/system/colors.ts (hunk 1 above)
Comment (jorgemoya) [reply packages/big-design-theme/src/system/colors.ts:56]: Might as well

**Pull request merged**